alpine-rails-buildkite/etc/sv/buildkite

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -o errexit -o pipefail -o noglob -o noclobber -o nounset
+IFS=$'\n\t'
+
+export HOME=/var/buildkite
+export GEM_HOME=${HOME}/.gem
+export BUILDKITE_AGENT_VERSION=edge
+export BUILDKITE_BUILD_PATH=/var/buildkite/builds
```

Comment:
Can you use `$HOME` to remove the redundant parts of the path here and on the next line?